### PR TITLE
Remote push type was removed from 5.x iOS SDK. 

### PR DESCRIPTION
**Pull request by dmipon**
Fix documentation accordingly.

### CODE REVIEW

**Comment by GiovanniBeriSinch** [comment]
There are a bunch of references to PushRemote in `voice-ios-cloud/reference`

```
~/workspace/docs/docs/voice/voice-ios-cloud/reference git:(fix/remote-push)
└─ rg -i remotepush
html/Protocols/SINClient.html
81:		<option value="//api/name/relayRemotePushNotification:">- relayRemotePushNotification:</option>
403:					<li><code><p>[SINClient relayRemotePushNotificationPayload:];</p></code></li>
573:	<a name="//api/name/relayRemotePushNotification:" title="relayRemotePushNotification:"></a>
574:	<h3 class="method-title"><code><a href="#//api/name/relayRemotePushNotification:">&ndash;&nbsp;relayRemotePushNotification:</a></code>
590:			<div class="method-subsection method-declaration"><code>- (id&lt;SINNotificationResult&gt;)relayRemotePushNotification:(NSDictionary *)<em>userInfo</em></code></div>

html/Protocols/SINCallNotificationResult.html
107:						<p>SINCallNotificationResult is used to indicate the result of [SINClient relayRemotePushNotificationPayload:]

html/Protocols/SINNotificationResult.html
107:						<p>SINNotificationResult is used to indicate the result of <a href="../Protocols/SINClient.html#//api/name/relayRemotePushNotification:">[SINClient relayRemotePushNotification:]</a> and
112:<p>id<SINNotificationResult> result = [self.client relayRemotePushNotification:payload];</p>
```

**Review by GiovanniBeriSinch — CHANGES_REQUESTED** [review]

File: docs/voice/voice-ios-cloud/voice-ios-cloud-push-notifications-callkit.md [hunk 1]
```diff
@@ -190,21 +191,7 @@ If the user of the application logs out or performs a similar action, the push n
...
-   "aps" : {
-      "alert" : {
-         "loc-key" : "SIN_INCOMING_CALL_DISPLAY_NAME",
-         "loc-args" : [ "John Doe"]
-      }
-   },
-   "sin": "<opaque push payload>"
-}
-```
+To support showing a user display name in a non-VoIP push notification (i.e. in a regular _Remote Push Notification_), you can configure a display name using [`-[SINManagedPush setDisplayName:]`].
```
[the remark this answers is not in the document]
Comment (dmipon) [reply docs/voice/voice-ios-cloud/voice-ios-cloud-push-notifications-callkit.md:194]: We still have it for VoIP

File: docs/voice/voice-ios-cloud/voice-ios-cloud-push-notifications-callkit.md [hunk 2]
```diff
@@ -52,12 +52,12 @@ iOS apps must have the proper entitlements to use push notifications. To add the
...
+> `SINManagedPush` is a very lightweight component and its life cycle can be independent of the life-cycle of a `SINClient`. It should be created once, and should not be disposed (that would prevent receiving push notifications via _PushKit_).
 
 
 ## Enabling Push Notifications for `SINClient`
 
-To enable that Sinch manage push notifications for you end-to-end, i.e. both client-side and server-side in terms acting [APNs Provider](https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/sending_notification_requests_to_apns?language=objc), it is necessary to enable that when configuring a `SINClient`.
+To make Sinch manage push notifications for you end-to-end, i.e. both client-side and server-side in terms acting [APNs Provider](https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/sending_notification_requests_to_apns?language=objc), it is necessary to enable that when configuring a `SINClient`.
```
[the remark this answers is not in the document]
Comment (dmipon) [reply docs/voice/voice-ios-cloud/voice-ios-cloud-push-notifications-callkit.md:60]: fixed

File: docs/voice/voice-ios-cloud/voice-ios-cloud-push-notifications-callkit.md [hunk 3]
```diff
@@ -52,12 +52,12 @@ iOS apps must have the proper entitlements to use push notifications. To add the
 
 > 👍
 >
-> `SINManagedPush` is a very lightweight component and its lifecycle can be independent of the life-cycle of a `SINClient`. It should be created once, and should not be disposed (that would prevent receiving push notifications via _PushKit_).
+> `SINManagedPush` is a very lightweight component and its life cycle can be independent of the life-cycle of a `SINClient`. It should be created once, and should not be disposed (that would prevent receiving push notifications via _PushKit_).
```
[the remark this answers is not in the document]
Comment (dmipon) [reply docs/voice/voice-ios-cloud/voice-ios-cloud-push-notifications-callkit.md:55]: fixed

File: docs/voice/voice-ios-cloud/voice-ios-cloud-push-notifications-callkit.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (GiovanniBeriSinch) [reply docs/voice/voice-ios-cloud/voice-ios-cloud-push-notifications-callkit.md:194]: > non-VoIP push notification (i.e. in a regular _Remote Push Notification_)

but this should be removed?

[the remark this answers is not in the document]
Comment (dmipon) [reply docs/voice/voice-ios-cloud/voice-ios-cloud-push-notifications-callkit.md:194]: Sorry, i misread it. Yes, that should be removed

[the remark this answers is not in the document]
Comment (dmipon) [reply docs/voice/voice-ios-cloud/voice-ios-cloud-push-notifications-callkit.md]: fixed

**Comment by GiovanniBeriSinch** [comment]
LGTM

**Pull request merged**